locales/en/allocations.json

```diff
@@ -24,5 +24,40 @@
   },
   "allocation_criteria": {
     "page_title": "Allocation criteria"
+  },
+  "view": {
+    "page_title": "View allocation",
+    "heading": "{{moves_count}} {{person_or_people}} from {{from_location}} to {{to_location}}, {{date}}",
+    "criteria_for_allocation": "Criteria for allocation",
+    "created_at": "Created {{datetime}}",
+    "allocation_tab": "Allocation",
```

Comment:
Maybe we can tidy up some of the keys that are no longer needed like these ones?